.github/workflows/build.yml

```diff
@@ -36,10 +36,13 @@ jobs:
                 pip install pyinstaller
 
             - name: PyInstaller (Terminal GUI)
-              run: pyinstaller --icon=images/icon.ico versions/terminal_gui.py
+              run: pyinstaller versions/terminal_gui.py --icon=images/icon.ico
 
             - name: PyInstaller (CLI)
-              run: pyinstaller --icon=images/icon.ico versions/cli_args.py
+              run: pyinstaller versions/cli_args.py --icon=images/icon.ico
+
+            - name: PyInstaller (GUI)
```

Comment:
Note to self: add this to the release workflow as well

Reply:
Extra note to self: Google Material Icon requires attribution and inclusion of license